lib/active_model/serializer/adapter/attributes.rb

```diff
@@ -0,0 +1,56 @@
+module ActiveModel
+  class Serializer
+    class Adapter
+      class Attributes < Adapter
```

Comment:
The duplicated code it bad :cry: but I know we need it in order to remove the inheritance.